x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -265,6 +267,23 @@ except if using `google` as provider. Required for providers: `default`, `azure`
 The client secret used as part of the authentication flow. It is always required
 except if using `google` as provider. Required for providers: `default`, `azure`.
 
+[float]
+==== `auth.oauth2.user`
+
+The user used as part of the authentication flow. It is required for authentication 
+- grant type password. It is always required except if using `google` as provider. 
```

Comment:
It doesn't look to be correct. The `user` option goes in pair with `password` for the default provider only. @kush-elastic would you mind read-proofing docs?